napari/settings/_base.py

```diff
@@ -358,14 +361,32 @@ def config_file_settings_source(
     """
     # _config_path is the primary config file on the model (the one to save to)
     config_path = getattr(settings, '_config_path', None)
-
+    if config_path:
+        config_path_Path = Path(config_path)
```

Comment:
Not sure I can stomach this variable name (casing of `Path` and repetition of path), though I appreciate we have to accommodate the existing `config_path` with type `Optional[str]`.

How about changing the definition of `config_path` to be something like:

```python
config_path = None
if hasattr(settings, '_config_path'):
    config_path = Path(settings._config_path)
```

then using `str(config_path)` when you need the string form?

Reply:
It looks like an old code. currently, settings have a `config_path` property that should always be defined.